insights/static/js/components/award-date-bar-chart.js

```diff
@@ -0,0 +1,174 @@
+const MS_IN_DAY = (1000 * 60 * 60 * 24); // number of milliseconds in a day
```

Comment:
![milliseconds in a day history](https://c.tadst.com/gfx/1200x675/length-of-day-graph.png)
by cosmic co-incidence, it actually is! :smile: